src/index.ts

```diff
@@ -56,11 +56,11 @@ export { pot };
 import * as fetch from "./fetch";
 export { fetch };
 
-import * as OptionEither from "./OptionEither";
-export { OptionEither };
+// import * as OptionEither from "./OptionEither";
+// export { OptionEither };
 
-import * as OptionValidation from "./OptionValidation";
-export { OptionValidation };
+// import * as OptionValidation from "./OptionValidation";
+// export { OptionValidation };
```

Comment:
I think those modules aren't much used in our codebase, let's think on just remove them